### DISCUSSION TITLE
ERCOT_PubAPI for Locations

### DISCUSSION THREAD

**Discussion by adidonatocda** [ercot/api-specs#36] [General]
I am working to build an application in the form of a graph with some geospatial mapping and time series charts. I cannot seem to find a way to extract geolocation information for nodal regions, generation locations, distribution locations, etc. Is there an API or endpoint with this type of data? Thanks in advance. -Anthony

**Comment by adidonatocda** [comment]
I found the information for generation: https://www.ercot.com/mp/data-products/data-product-details?id=pg7-200-er

**Reply by ajay-ercot** [reply]
Hello, sorry we currently don't have any public API's that can offer GIS co-ordinates for the network model information like price nodes, synchronized generators, distribution substations, etc.

**Reply by p0w3rman** [reply]
Is there a different way to find this - does ERCOT publish a node -> lat/long list?

**Comment by CaffeineLab** [comment]
You kind of have to piece everything together from public resources.  I was working on a similar project and kept banging up against this.  For some reason the info you want is held close and never published.  I mean - it's on their maps as points... but you can't use those in your project ... as of today I have never found a published list anywhere.    

Some Googling gets you things like:

https://en.wikipedia.org/wiki/List_of_power_stations_in_Texas
https://globalenergyobservatory.org/
https://eerscmap.usgs.gov/uswtdb/
https://www.thewindpower.net/ (similar)
https://globalenergyobservatory.org/index.php
